src/coreclr/jit/emit.h

```diff
@@ -1534,13 +1534,15 @@ class emitter
             BYTE* idjAddr; // address of jump ins (for patching)
         } idjTemp;
 
-        unsigned idjOffs : 30;    // Before jump emission, this is the byte offset within IG of the jump instruction.
+        unsigned idjOffs : 29;    // Before jump emission, this is the byte offset within IG of the jump instruction.
                                   // After emission, for forward jumps, this is the target offset -- in bytes from the
                                   // beginning of the function -- of the target instruction of the jump, used to
                                   // determine if this jump needs to be patched.
         unsigned idjShort : 1;    // is the jump known to be a short  one?
         unsigned idjKeepLong : 1; // should the jump be kept long? (used for
                                   // hot to cold and cold to hot jumps)
+        unsigned idjIsJmpAlways : 1; // indicates that the jump was added at the end of a BB_ALWAYS basic block
+                                     // in case the next block was not the next target
```

Comment:
```suggestion
        unsigned idjIsJmpAlways : 1; // indicates that the jump was added at the end of a BBJ_ALWAYS basic block
                                     // in case the next block was not the next target
```

This comment confuses me. What is the "next target"?

Reply:
I've removed the second line. So it's now just `/ indicates that the jump was added at the end of a BBJ_ALWAYS basic block` and anyone interested can lookup the logic used around BBJ_ALWAYS for the details on the logic surrounding it. If the name changes this comment will change along with it.